### PR TITLE
Add kube-probe to capture Kubernetes HTTP probes

### PR DESCRIPTION
**Pull request by joelzwarrington**
kubernetes has [HTTP probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes) to check readiness of a service

Before:
```ruby
Browser.new('kube-probe/1.25').bot?
=> false
```

After:
```ruby
Browser.new('kube-probe/1.25').bot?
=> true
```

### CODE REVIEW

**Comment by fnando** [comment]
Hi @joelzwarrington!

Can you please follow the [other instructions listed here](https://github.com/fnando/browser#adding-a-new-bot)?

File: bots.yml [hunk 1]
```diff
@@ -125,6 +125,7 @@ knows.is: knows.is
 kojitsubot: Kojitsubot
 komodiabot: KomodiaBot
 kraken: kraken
+kube-probe: Kubernetes HTTP Probe
```
Comment (kuldeepaggarwal) [inline bots.yml:128]: ❤️

**Comment by joelzwarrington** [comment]
@fnando should be good to have a look now, I believe I followed all of the instructions 😄

**Comment by joelzwarrington** [comment]
@fnando any chance you could have a look? 🙇 🙏

**Comment by joelzwarrington** [comment]
@fnando - can you have a look please?

**Comment by joelzwarrington** [comment]
@fnando can you have a look please?

**Pull request closed without merging**